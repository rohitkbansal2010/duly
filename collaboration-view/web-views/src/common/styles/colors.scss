//Primary
$dupage-dark-blue: #002855;

//Secondary
$dupage-light-blue: #00a5df;
$dupage-light-blue-tint: #c7edff;
$accent-blue: #345aa3;
$dupage-blue: #00b2f0;
$dupage-icon-bg: #00b2f04d;

//Tertiary Pairs
$dupage-violet: #470a68;
$dupage-magenta: #c5299b;
$dupage-magenta-tint: #ffdaf5;
$dupage-orange: #ff6a14;
$dupage-saffron: #f2a900;
$dupage-deep-green: #004c45;
$dupage-green: #78be21;

//Tints with Opacity
$magenta-tint-opacity: #ffecfa;
$green-tint-opacity: #ebf6df;
$saffron-tint-opacity: #fff3d7;
$medium-gray-opacity: #d3d7dc;

//Text
$default-text: #0f172a;
$white: #ffffff;

//Grays
$alto-gray: #e0e0e0;
$dark-gray: #666e7d;
$dove-gray: #666666;
$medium-gray: #a8afb9;
$gray: #d6dbe0;
$light-gray: #eceef0;
$ghost-gray: #f3f6fa;
$mercury: #e5e5e5;
$chateau-gray: #a8afb9;
$gmail-black: #292929;
$gmail-grey: #5d5c5d;
$light-gray: #f4f7fa;

//Blue
$alice-blue: #fafdff;
$cornflower-blue: #bdbdbd;
$pattens-blue: #e0f0ff;
$cerulean-blue: #0da3d8;
$purple-heart-blue: #6140cb;
$light-blue: #aadef2;

//Reds
$dark-red: #be123c;
$medium-red-violet: #c5299b;
$light-violet: #ecbde0;

//Purple
$medium-purple: #6761dc;

//Alerts
$green-success: #00813b;
$yellow-warning: #fad306;
$red-error: #9f1239;

//Miscellaneous
$abbey: #4d5057;
$silver: #cccccc;
$light-silver: #c4c4c4;
$black: #000000;
$dark-cerulean: #1c4380;
$mabel: #d5f1ff;
$negroni: #ffdbc5;
$light-pink: #ede7f0;
$iron: #d4d7db;

//Tints
$magenta-tint: #ffdaf5;
$saffron-tint: #ffe7b0;
$deep-green-tint: #c1e4e1;
$green-tint: #d8edbe;
$orange-tint: #ffdbc5;
$violet-tint: #e3cdee;
$dark-blue-tint: #e5edfd;

//Legend
$normal-range: #f6f2f7;
$concerning-range: #eee6f1;
$low-range: #e7dfeb;
$brain-affecting-range: #ddd1e3;
