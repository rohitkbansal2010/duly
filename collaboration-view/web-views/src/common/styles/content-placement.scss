@import "devices";

//Landscape width
$content-width-landscape: $landscape;
$content-px-landscape: 70px;
$content-mt-negative-landscape: -48px;
$content-mt-positive-landscape: 48px;

//Mobile plus width
$content-px-mobile-plus: 27px;

//Mixins
@mixin content-padding-landscape {
  max-width: $content-width-landscape !important;
  padding: 0 $content-px-landscape !important;
}

@mixin content-padding-mobile-plus {
  @media (max-width: $mobile-plus) {
    padding: 0 $content-px-mobile-plus !important;
  }
}
