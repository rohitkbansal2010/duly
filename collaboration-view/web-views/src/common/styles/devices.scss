$mobile: 375px;
$mobile-plus: 440px;
$portrait: 1024px;
$landscape: 1366px;
$desktop: 1440px;
