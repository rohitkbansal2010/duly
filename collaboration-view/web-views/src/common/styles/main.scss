@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;700;800&display=swap");
@import url("https://fonts.googleapis.com/css2?family=Lora:wght@100;200;400;700;800&display=swap");
@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;600;700;800&display=swap");
@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;200;400;500;700;800&display=swap");

@import "~@styles/devices.scss";
@import "~@styles/colors.scss";

@import "~@styles/media-mixins.scss";

@font-face {
  font-weight: 900;
  font-family: "NexaHeavy";
  src: url("../fonts/Nexa/Nexa-Heavy.otf") format("opentype");
}

@font-face {
  font-weight: 700;
  font-family: "NexaExtraBold";
  src: url("../fonts/Nexa/Nexa-ExtraBold.otf") format("opentype");
}

@font-face {
  font-weight: 600;
  font-family: "NexaBold";
  src: url("../fonts/Nexa/Nexa-Bold.otf") format("opentype");
}

@font-face {
  font-weight: 400;
  font-family: "NexaRegular";
  src: url("../fonts/Nexa/Nexa-Regular.otf") format("opentype");
}

html,
body {
  position: fixed;
  width: 100%;
  height: 100%;
  overflow-y: hidden;
  font-size: 1.25vh !important;
  font-family: "NexaRegular", sans-serif !important;
  background-color: $ghost-gray !important;

  &.modal-open {
    padding-right: 0 !important;
    overflow: auto !important;
  }
}

:global(body.modal-open > :not(.modal)) {
  filter: blur(0.5rem);
}

:global(.swiper-slide-active) {
  z-index: 2 !important;
}

:global(#root) {
  position: fixed;
  display: flex;
  flex-flow: column nowrap;
  width: 100%;
  height: 100%;
  overflow-y: hidden;
  font-family: "NexaRegular", sans-serif;
}

@mixin title-text {
  font-weight: 700;
  font-size: 1.125rem;
  line-height: 24px;
}

@mixin normal-text {
  font-weight: 400;
  font-size: 0.875rem;
  line-height: 24px;
}

@mixin thin-text {
  font-weight: 200;
  line-height: 24px;
}

@mixin card-border {
  border-color: $gray;
  border-width: 1px;
  border-radius: 8px;
}

@mixin card-shadow($color, $inset) {
  box-shadow: 0 0 1px $light-gray, 0 10px 15px -3px $light-gray, 0 4px 6px -2px $light-gray,
    260px 0 0 -244px $color $inset;
}

@mixin remove-focus($class) {
  #{$class}:focus {
    outline: 0 !important;
    box-shadow: none !important;
  }
}

@function convertToRGBA($color, $opacity) {
  @return rgba(red($color), green($color), blue($color), $opacity);
}

@mixin background-opacity($color, $opacity) {
  background-color: convertToRGBA($color, $opacity);
}

@mixin ios-scrollbar {
  & {
    overflow-y: scroll;
  }
  &::-webkit-scrollbar {
    width: 1rem;
    background-color: $white;
  }

  &::-webkit-scrollbar-track {
    background-color: inherit;
  }

  &::-webkit-scrollbar-thumb {
    background-color: #babac0;
    border: 0.25rem solid $white;
    border-radius: 1rem;
  }

  &::-webkit-scrollbar-button {
    display: none;
  }
}

@mixin invisible-scroll {
  position: relative;
  overflow-y: scroll;
  -ms-overflow-style: none;
  scrollbar-width: none;

  &::-webkit-scrollbar {
    display: none;
  }
}

@mixin invisible-scroll-top-fade {
  position: absolute;
  top: 0;
  right: 0;
  left: 0;
  z-index: 10;
  height: 6rem;
  background-image: linear-gradient($ghost-gray, transparent);
  opacity: 1;
  transition: opacity 0.3s ease;
  pointer-events: none;
}

@mixin invisible-scroll-bottom-fade {
  position: absolute;
  right: 0;
  bottom: 0;
  left: 0;
  z-index: 10;
  height: 8rem;
  background-image: linear-gradient(transparent, $ghost-gray);
  opacity: 1;
  transition: opacity 0.3s ease;
  pointer-events: none;
}
