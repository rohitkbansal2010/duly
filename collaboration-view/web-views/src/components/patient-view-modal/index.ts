export { PatientViewModal } from './patient-view-modal';
