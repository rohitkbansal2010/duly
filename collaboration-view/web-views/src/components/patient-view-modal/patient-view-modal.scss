@import "~@styles/themes.scss";
@import "~@styles/typography.scss";

.patientViewModal {
  &Header {
    justify-content: center !important;
    margin-bottom: 4.25rem;
    padding: 0 !important;
    border: 0 !important;

    & img {
      width: 12.75rem;
      height: 5.94rem;
    }
  }

  &Body {
    padding: 0 !important;

    &Avatar {
      display: flex !important;
      align-items: center !important;
      justify-content: center !important;
      width: 13.3rem;
      height: 13.3rem;

      margin: 0 auto;
      background: $white;
      border-radius: 50%;
    }

    &HumanName {
      @include nexa-heavy-highlight;

      margin-top: 2rem;
      color: $white;
      font-weight: 800;
      font-size: 2.5rem;
      line-height: 3rem;
    }

    &AdditionalInfo {
      @include nexa-regular-highlight;

      margin-top: 2rem;
      color: $white;
      font-weight: 400;
      font-size: 1.5rem;
      line-height: 2rem;
      text-transform: capitalize;
    }

    &NextSteps {
      margin-top: 6rem;
      &ButtonPrimary {
        width: 26.9375rem !important;
        height: 4.625rem !important;
        color: $white;
        font-weight: 800;
        font-size: 1.25rem !important;
        line-height: 1.5rem !important;
        background-color: $dupage-dark-blue !important;
        border-color: $dupage-dark-blue !important;
        border-radius: 0.5rem;
      }
      &ButtonDefault {
        width: 26.9375rem !important;
        height: 4.625rem !important;
        margin-top: 4rem;
        color: $white;
        font-weight: 800;
        font-size: 1.25rem !important;
        line-height: 1.5rem !important;
        background-color: transparent !important;
        border: 1px solid $dupage-dark-blue !important;
        border-radius: 0.5rem;
      }
    }
  }
}
