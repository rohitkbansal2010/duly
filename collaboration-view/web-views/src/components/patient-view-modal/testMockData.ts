export const mockUtilsReturnValues = { 
  getSrcAvatar: 'mockSrc', 
  calculateYears: 100, 
  writeHumanName: 'mockWriteHumanName', 
};

export const mockStyles = {
  patientViewModalHeader: 'mockClassNameHeader',
  patientViewModalBody: 'mockClassNameBody',
  patientViewModalBodyAvatar: 'mockClassNameAvatar',
  patientViewModalBodyHumanName: 'mockClassNameHumanName',
  patientViewModalBodyAdditionalInfo: 'mockClassNameAdditionalInfo',
};

export const mockConstants = { PATIENT_VIEW_MODAL_AVATAR_SIZE: 50 };

export interface MockAvatarProps {
	width: number;
	src: string;
	role: string;
	hasBorder: boolean;
}

export const mockAvatarProps: MockAvatarProps = {
  width: 50,
  src: mockUtilsReturnValues.getSrcAvatar, 
  role: 'mockRole', 
  hasBorder: true, 
};
