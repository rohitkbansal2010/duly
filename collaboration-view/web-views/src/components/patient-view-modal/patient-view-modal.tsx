
import { Modal } from 'react-bootstrap';
import { useSelector } from 'react-redux';
import { useParams, useHistory } from 'react-router-dom';

import { Avatar } from '@components/avatar';
import { SideModal } from '@components/side-modal';
import { PATIENT_VIEW_MODAL_AVATAR_SIZE } from '@constants';
import { AvatarByRole } from '@enums';
import { usePatientViewModal, useAzureADAuth } from '@hooks';
import { dulyLargeWhiteDarkBlueIcon } from '@icons';
import { RootState } from '@redux/reducers';
import { PatientData } from '@types';
import { calculateYears, getSrcAvatar, writeHumanName } from '@utils';

import styles from './patient-view-modal.scss';

type AppointmentParams = {
  appointmentId: string;
  patientId: string;
  practitionerId: string;
};
export const PatientViewModal = () => {
  const history = useHistory();
  const { logoutAzureADRedirect } = useAzureADAuth();
  const { appointmentId, patientId, practitionerId } = useParams<AppointmentParams>();

  const { isShowPatientViewModal, onHidePatientViewModal } = usePatientViewModal();
  const patientData: PatientData | null = useSelector(
    ({ CURRENT_APPOINTMENT }: RootState) =>
      CURRENT_APPOINTMENT.patientData
  );

  const givenNames: string[] = patientData?.generalInfo?.humanName?.givenNames || [];
  const familyName: string = patientData?.generalInfo?.humanName?.familyName || '';
  const gender: string = patientData?.gender || '';
  const birthDate: string | undefined = patientData?.birthDate;
  const yearsOld: number | undefined = calculateYears(birthDate);

  const StartCheckout = () => {
    sessionStorage.removeItem('metric'); // set toggle to off when logout.
    logoutAzureADRedirect(`/start-checkout/${appointmentId}/${patientId}/${practitionerId}`);
  };

  const gotoAfterSteps = () => {
    history.push(`/exam-room/schedule-follow-up/${appointmentId}/${patientId}/${practitionerId}/checkout`);
  };
  return (
    <SideModal show={isShowPatientViewModal} onHide={onHidePatientViewModal}>
      <Modal.Header className={styles.patientViewModalHeader}>
        <img src={dulyLargeWhiteDarkBlueIcon} alt="Large duly icon" />
      </Modal.Header>
      <Modal.Body className={styles.patientViewModalBody}>
        <div className={styles.patientViewModalBodyAvatar}>
          <Avatar
            width={PATIENT_VIEW_MODAL_AVATAR_SIZE}
            src={getSrcAvatar(patientData?.photo)}
            role={AvatarByRole.NON_EMPLOYEE}
            hasBorder
          />
        </div>
        <div className={styles.patientViewModalBodyHumanName} data-testid="humanName">
          {writeHumanName(givenNames, familyName)}
        </div>
        <div className={styles.patientViewModalBodyAdditionalInfo}>
          <div data-testid="gender">{gender}</div>
          <div data-testid="age">{!!yearsOld && `${yearsOld} years old`}</div>
        </div>
        <div className={styles.patientViewModalBodyNextSteps}>
          <div>
            <button
              className={styles.patientViewModalBodyNextStepsButtonPrimary}
              onClick={() => 
                StartCheckout()}
              data-automation="handover-to-ma-button"
            >
              Handover to MA
            </button>
            <button
              className={styles.patientViewModalBodyNextStepsButtonDefault}
              onClick={() =>
                {gotoAfterSteps();
                onHidePatientViewModal();}
              }
              data-automation="continue-to-after-visit-steps-button"

            >
              Continue to After Visit Steps
            </button>
          </div>
        </div>
      </Modal.Body>
    </SideModal>
  );
};
