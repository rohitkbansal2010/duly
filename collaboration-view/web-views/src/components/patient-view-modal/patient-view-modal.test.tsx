import { mount, ReactWrapper } from 'enzyme';
import toJson from 'enzyme-to-json';
import { ReactChild } from 'react';
import { useSelector } from 'react-redux';

import { patientMockData } from '@mock-data';
import {
  calculateYears,
  getSrcAvatar,
  writeHumanName
} from '@utils';

import { PatientViewModal } from './patient-view-modal';
import styles from './patient-view-modal.scss';
import { MockAvatarProps } from './testMockData';

jest.mock('./patient-view-modal.scss', () => 
  require('./testMockData').mockStyles);

jest.mock('@constants', () => 
  require('./testMockData').mockConstants);

jest.mock('@enums', () => {
  const originalModule = jest.requireActual('@enums');

  return {
    __esModule: true,
    ...originalModule,
    AvatarByRole: { NON_EMPLOYEE: 'mockRole' }, 
  };
});

jest.mock('react-redux', () => 
  ({ useSelector: jest.fn() }));

  
jest.mock('react-router-dom', () => 
  ({
    ...jest.requireActual('react-router-dom'), // use actual for all non-hook parts
    useParams: () => 
      ({
        appointmentId: 'appointment-id',
        patientId: 'patient-id',
        practitionerId: 'practitioner-id',
      }),
  }));

jest.mock('@utils', () => 
  ({
    getSrcAvatar: jest.fn(),
    calculateYears: jest.fn(),
    writeHumanName: jest.fn(),
  }));
	
const mockUsePatientViewModal = jest.fn();
const mockLogoutAzureADRedirect = jest.fn();
jest.mock('@hooks', () => 
  ({
    useAzureADAuth: () =>
      ({ logoutAzureADRedirect: mockLogoutAzureADRedirect }),
    usePatientViewModal: () => 
      ({
        onHidePatientViewModal: mockUsePatientViewModal,
        isShowPatientViewModal: true, 
      }), 
  }));

const mockSideModal = jest.fn();
interface MockSideModalProps {
	show: boolean;
	onHide: () => void;
	children: ReactChild
}
jest.mock('@components/side-modal', () => 
  ({
    SideModal: (props: MockSideModalProps) => {
      mockSideModal(props);
      return props.children;
    }, 
  }));

const mockModalHeader = jest.fn();
const mockModalBody = jest.fn();
interface MockModalBSProps {
	className: string;
	children: ReactChild
}
jest.mock('react-bootstrap', () => 
  ({
    Modal: {
      Header: (props: MockModalBSProps) => {
        mockModalHeader(props);
        return props.children;
      },
      Body: (props: MockModalBSProps) => {
        mockModalBody(props);
        return props.children;
      },
    },
  }));

const mockAvatar = jest.fn();
jest.mock('@components/avatar', () => 
  ({
    Avatar: (props: MockAvatarProps) => {
      mockAvatar(props);
      return null;
    }, 
  }));

describe('PatientViewModal with normal data', () => {
  let wrapper: ReactWrapper;
  beforeEach(() => {
    (useSelector as jest.Mock).mockImplementationOnce(() => 
      patientMockData);

    const { mockUtilsReturnValues } = require('./testMockData');
    (getSrcAvatar as jest.Mock).mockImplementationOnce(() => 
      mockUtilsReturnValues.getSrcAvatar);
    (calculateYears as jest.Mock).mockImplementationOnce(() => 
      mockUtilsReturnValues.calculateYears);
    (writeHumanName as jest.Mock).mockImplementationOnce(() => 
      mockUtilsReturnValues.writeHumanName);
		
    wrapper = mount(<PatientViewModal />);
  });

  it('should match snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should pass correct show and onHide props in nested SideModal', () => {
    expect(mockSideModal).toHaveBeenCalledWith(
      expect.objectContaining({
        show: true,
        onHide: mockUsePatientViewModal,
      })
    );
  });

  it('should pass correct className prop in nested Modal.Header', () => {
    expect(mockModalHeader).toHaveBeenCalledWith(
      expect.objectContaining({ className: styles.patientViewModalHeader })
    );
  });

  it('should pass correct className prop in nested Modal.Body', () => {
    expect(mockModalBody).toHaveBeenCalledWith(
      expect.objectContaining({ className: styles.patientViewModalBody })
    );
  });

  it('should pass correct width, src, role and hasBorder props in nested Avatar', () => {
    const { mockAvatarProps } = require('./testMockData');

    expect(mockAvatar).toHaveBeenCalledWith(
      expect.objectContaining(mockAvatarProps)
    );
  });

  it('should pass photoData into getSrcAvatar', () => {
    expect(getSrcAvatar).toHaveBeenCalledWith(patientMockData.photo);
  });

  it('should pass givenNames and familyName into writeHumanName', () => {
    const { givenNames, familyName } = patientMockData.generalInfo.humanName;

    expect(writeHumanName).toHaveBeenCalledWith(givenNames, familyName);
  });

  it('should render correct name in specific element', () => {
    const text = wrapper.find({ 'data-testid': 'humanName' }).text();
    const mockHumanName = require('./testMockData').mockUtilsReturnValues.writeHumanName;
		
    expect(text).toEqual(mockHumanName);
  });

  it('should render correct gender in specific element', () => {
    const elementText = wrapper.find({ 'data-testid': 'gender' }).text();
		
    expect(elementText).toEqual(patientMockData.gender);
  });

  it('should pass birthDate into calculateYears', () => {
    expect(calculateYears).toHaveBeenCalledWith(patientMockData.birthDate);
  });

  it('should render correct age in specific element', () => {
    const elementText = wrapper.find({ 'data-testid': 'age' }).text();
    const mockAge = require('./testMockData').mockUtilsReturnValues.calculateYears;
		
    expect(elementText).toEqual(`${mockAge} years old`);
  });
});

describe('PatientViewModal with undefined data', () => {
  let wrapper: ReactWrapper;
  beforeEach(() => {
    (useSelector as jest.Mock).mockImplementationOnce(() => 
      ({ patientMockData }));
		
    wrapper = mount(<PatientViewModal />);
  });
	
  it('should pass empty string in src prop in nested Avatar', () => {
    (getSrcAvatar as jest.Mock).mockReturnValueOnce('');
    mount(<PatientViewModal />);

    expect(mockAvatar).toHaveBeenCalledWith(
      expect.objectContaining({ src: '' })
    );
  });

  it('should pass undefined instead of photoData into getSrcAvatar', () => {
    expect(getSrcAvatar).toHaveBeenCalledWith(undefined);
  });

  it('should pass ([], "") into writeHumanName if there is no givenNames and familyName', () => {
    expect(writeHumanName).toHaveBeenCalledWith([], '');
  });

  it('should render nothing in specific element if there is no gender', () => {
    const elementText = wrapper.find({ 'data-testid': 'gender' }).text();
		
    expect(elementText).toEqual('');
  });

  it('should render nothing in specific element if there is no birthDate', () => {
    const elementText = wrapper.find({ 'data-testid': 'age' }).text();

    expect(elementText).toEqual('');
  });
});
