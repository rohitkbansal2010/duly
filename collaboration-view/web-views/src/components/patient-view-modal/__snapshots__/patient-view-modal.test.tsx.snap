// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PatientViewModal with normal data should match snapshot 1`] = `
<PatientViewModal>
  <SideModal
    onHide={[MockFunction]}
    show={true}
  >
    <Header
      className="mockClassNameHeader"
    >
      <img
        alt="Large duly icon"
        src="duly-large-white-dark-blue.svg"
      />
    </Header>
    <Body
      className="mockClassNameBody"
    >
      <div
        className="mockClassNameAvatar"
      >
        <Avatar
          hasBorder={true}
          role="mockRole"
          src="mockSrc"
          width={50}
        />
      </div>
      <div
        className="mockClassNameHumanName"
        data-testid="humanName"
      >
        mockWriteHumanName
      </div>
      <div
        className="mockClassNameAdditionalInfo"
      >
        <div
          data-testid="gender"
        >
          Female
        </div>
        <div
          data-testid="age"
        >
          100 years old
        </div>
      </div>
      <div>
        <div>
          <button
            data-automation="handover-to-ma-button"
            onClick={[Function]}
          >
            Handover to MA
          </button>
          <button
            data-automation="continue-to-after-visit-steps-button"
            onClick={[Function]}
          >
            Continue to After Visit Steps
          </button>
        </div>
      </div>
    </Body>
  </SideModal>
</PatientViewModal>
`;
